server/api/router.go

```diff
@@ -80,7 +80,9 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	router.Handle("/api/v1/status", newStatusHandler(rd)).Methods("GET")
 
 	router.Handle("/api/v1/members", newMemberListHandler(svr, rd)).Methods("GET")
-	router.Handle("/api/v1/members/{name}", newMemberDeleteHandler(svr, rd)).Methods("DELETE")
+	memberDeleteHandler := newMemberDeleteHandler(svr, rd)
+	router.HandleFunc("/api/v1/members/{name}", memberDeleteHandler.DeleteByName).Methods("DELETE")
```

Comment:
Can't find a best practice after a simple search, but I tend to agree with `members/name/{name}` and `members/id/{id}` now.
Ref: https://stackoverflow.com/questions/842261/rest-supporting-multiple-possible-identifiers